dataland-backend/src/test/kotlin/org/dataland/datalandbackend/services/DataOwnersManagerTest.kt

```diff
@@ -76,4 +92,58 @@ class DataOwnersManagerTest {
         }
         assertTrue(exception.summary.contains("User is already a data owner for company."))
     }
+
+    @Test
+    fun `check that email for users becoming company data owner is not generated if company does not exist`() {
+        val mockStoredCompany = mock(StoredCompanyEntity::class.java)
+
```

Comment:
do you really need the mockStoredCompany?
isnt it possible to just use the following?
dataOwnersManager.addDataOwnerToCompany(
                companyId = UUID.ranodm().toString(),
                userId = testUserId,
                companyName = testCompanyName,
            )

Reply:
right, it is possible. It is revised.